<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #111111</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="mazeGraphicView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>831</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delayLineEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Renogare</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #cccccc;
	color:  #1a1a1a;
	border-radius: 5px;
 	border: 2px solid #222222;
	padding: 0 0 2px
}

QLineEdit:hover {
	background-color: #eeeeee;
 	border: 2px solid #666666;
}
QLineEdit:focus {
	background-color: #eeeeee;
 	border: 2px solid #666666;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="delayLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>205</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Renogare</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: #eee;
</string>
    </property>
    <property name="text">
     <string>Délai en millisecondes :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mazeRowsLineEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Renogare</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #cccccc;
	color:  #1a1a1a;
	border-radius: 5px;
 	border: 2px solid #222222;
	padding: 0 0 2px
}

QLineEdit:hover {
	background-color: #eeeeee;
 	border: 2px solid #666666;
}
QLineEdit:focus {
	background-color: #eeeeee;
 	border: 2px solid #666666;}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Renogare</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cccccc;
	color:  #1a1a1a;
	border-radius: 5px;
 	border: 2px solid #222222;
}

QPushButton:hover {
	background-color: #eeeeee;
 	border: 2px solid #666666;
}
</string>
    </property>
    <property name="text">
     <string>Créer</string>
    </property>
   </widget>
   <widget class="QLabel" name="mazeRowsLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Renogare</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: #eee;
</string>
    </property>
    <property name="text">
     <string>Nombre de Colones :</string>
    </property>
   </widget>
   <widget class="QLabel" name="leaveLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Renogare</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: #eee;
</string>
    </property>
    <property name="text">
     <string>Quitter [échap]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
